### DISCUSSION TITLE
SixLabors.ImageSharp.Image.Load<Rgba32>(path)

### DISCUSSION THREAD

**Discussion by DungHD-1997** [SixLabors/ImageSharp#2574] [Q&A]
![image](https://github.com/SixLabors/ImageSharp/assets/119638167/25bb501d-af39-4e2e-a650-b6c4e29ebabd)
I used the Load function to read an image with width = 4656, height= 3496, size = 1.4Mb, the problem I encountered is that the Load function executes very slowly. SixLabors.ImageSharp version 2.1.6 and Net Framework 4.7.2.
The code is as shown above, and the execution time is as shown below. Please show me how to load images faster. thank you.
![image](https://github.com/SixLabors/ImageSharp/assets/119638167/5350cffb-e188-4d6f-a5ca-b8618691dc19)

**Comment by JimBobSquarePants** [comment]
You’ve provided nothing by which we can help you. You’re also using stopwatch to benchmark the application with no warmup which is worse than useless. You should be using BenchmarkDotnet

You’re using an old major version on an old target framework (not even the most recent of the legacy types) that lacks proper SIMD intrinsic support. 

You want to go faster. Move to .NET 6+ and use v3+.

**Reply by DungHD-1997** [reply]
sorry this is my fault, thank you for your reply.
